ui/src/app/views/workspace-nav-bar/component.css

```diff
@@ -1,138 +1,6 @@
-  width: 8rem;
-  background-color: transparent;
-  border: none;
-  border-radius: 0;
-  padding: 0;
-}
-
-::ng-deep #workspace-top-nav-bar .btn.btn-primary.btn-top:hover {
-  background-color: rgba(255, 255, 255, 0.15);
-}
-
-::ng-deep #workspace-top-nav-bar .btn.btn-primary.btn-top:disabled {
-  cursor: not-allowed;
-  background-color: transparent;
-}
-
-::ng-deep #workspace-top-nav-bar .btn-group-overflow .btn.dropdown-toggle {
-  height: 100%;
-  width: 2rem;
-  background-color: transparent;
-  border-left: 1px solid rgba(255, 255, 255, 0.15);
-  border-right: 1px solid rgba(255, 255, 255, 0.15);
-  border-radius: 0;
-  color: white;
-  border-top: none;
-  border-bottom: none;
-  position: absolute;
-  right: 0;
-}
-
-::ng-deep #workspace-top-nav-bar .btn-group-overflow .btn.dropdown-toggle:hover {
-  background-color: rgba(255, 255, 255, 0.15);
-}
-
-::ng-deep #workspace-top-nav-bar btn.btn-primary.btn-top.active {
-  background-color: rgba(255, 255, 255, 0.15);
-}
-
-:host::ng-deep #workspace-top-nav-bar .btn.btn-primary.btn-top.btn-dropdown {
-  color: white;
-  width: 2rem;
-  border: none;
-  height: 100%;
-  position: absolute;
-  right: 0;
-  padding: 0;
-  overflow: visible;
-  outline: none;
-}
-
-:host::ng-deep .nav-bar-dropdown.dropdown .dropdown-toggle:not(.btn) {
-  color: white;
-  padding: 0;
-  outline: none;
-  height: 100%;
-  width: 100%;
-}
-
-:host::ng-deep clr-dropdown.nav-bar-dropdown.dropdown {
-  height: 100%;
-  width: 100%;
-}
-
-::ng-deep #workspace-top-nav-bar .btn-top.btn-overflow:hover {
-  background-color: rgba(255, 255, 255, 0.15);
-}
-/*
- * The use of !important here is to prevent clarity from overriding the position.
- * Clarity's overflow elements apply a static position on click, so we needed to
- * override that.
- */
-:host::ng-deep #workspace-top-nav-bar.btn-group>.btn-group-overflow {
-  position: absolute !important;
-  right: 0;
-  height: 100%;
-  width: 5rem
-}
-
-:host::ng-deep #workspace-top-nav-bar .dropdown-menu {
-  width: 8rem;
-  position: absolute;
-  /*
-   * The important labels are necessary to not accidentally create space off the
-   * edge of the page
-   */
-  left: -120px !important;
-  top: 55px !important;
-}
-
-.tab-inner {
-  height: 100%;
-  line-height: 55px;
-  width: 100%;
-}
-
-.selected-tab-inner {
-  background-color: rgba(255,255,255,0.15);
-  border-bottom: 8px solid #2691D0;
-  font-weight: bold;
-}
-
-label.dropdown-header {
-  font-size: 14px;
-  color: #262262;
-  font-weight: 600;
-  text-transform: none;
-}
-
-.btn-group {
-  /*Account for margins.*/
-  width: calc(100% + 1.6rem);
-  background-color: #5FAEE0;
-  height: 60px;
-  margin-left: -1rem;
-  position: relative;
-  margin-right: 0;
-  border-bottom: 5px solid #2691D0;
-  padding-left: 5rem;
-  box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.26), 0 2px 10px 0 rgba(0, 0, 0, 0.16);
-}
-
-.btn-group .dropdown-menu clr-icon {
-  display: inline-grid;
-  margin-left: -0.4rem;
-}
-
 @media print {
```

Comment:
Do we really expect anyone to print the page? Can this file be pulled entirely?

Reply:
On further research, it appears this was intentional to allow for printing of one of the nav-bar'd pages (the cohort report). https://github.com/all-of-us/workbench/pull/1599

Reply:
We should probably have a global definition for that class then, yes? I've seen it in a few other places

Reply:
Yep, that sounds reasonable. Probably not worth breaking per-component CSS encapsulation in Angular, but given our current React setup it makes sense.